### PR TITLE
Remove locking function from the list of unsupported functions

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Remove locking function from the list of unsupported functions.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Mini256** [comment]
/uncc Mini256
/cc qw4990

I am not familiar with this field, @qw4990, could you make a confirmation about it?

File: functions-and-operators/miscellaneous-functions.md [hunk 1]
```diff
@@ -30,7 +30,5 @@ TiDB 支持使用 MySQL 5.7 中提供的大部分[其他函数](https://dev.mysq
 
 | 函数名 | 功能描述  |
 |:------|:-----------|
-| [`GET_LOCK()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_get-lock)  | 获取命名锁，详见 [TiDB #10929](https://github.com/pingcap/tidb/issues/14994) |
-| [`RELEASE_LOCK()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_release-lock)  | 释放命名锁，详见 [TiDB #10929](https://github.com/pingcap/tidb/issues/14994)  |
 | [`UUID_SHORT()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_uuid-short)  | 基于特定假设提供唯一的 UUID，目前这些假设在 TiDB 中不存在，详见 [TiDB #4620](https://github.com/pingcap/tidb/issues/4620) |
-| [`MASTER_WAIT_POS()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_master-pos-wait)  | 与 MySQL 同步相关 |
\ No newline at end of file
+| [`MASTER_WAIT_POS()`](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_master-pos-wait)  | 与 MySQL 同步相关 |
```
Comment (shichun-0415) [inline functions-and-operators/miscellaneous-functions.md:34]: The parameter in the link is displayed as `MASTER_POS_WAIT`.

Comment (TomShawn) [reply functions-and-operators/miscellaneous-functions.md:34]: Thanks for pointing it out. I will address the issue in another PR.

**Review by CbcWestwolf — APPROVED** [review]

**Pull request merged**